.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -207,20 +110,13 @@ jobs:
         with:
           name: webdriverio-screenshots
           path: /home/runner/work/oppia/webdriverio-screenshots
-      - name: Uploading webpack bundles as an artifact
-        if: ${{ failure() }}
-        uses: actions/upload-artifact@v3
-        with:
-          name: webpack-bundles
-          path: /home/runner/work/oppia/oppia/build
```

Comment:
Can we keep this? It was added to help debug test failures

Reply:
I updated the issue to have that information. When we make the change from dev mode to prod mode we will include this feature. https://github.com/oppia/oppia/issues/19096